src/util/loc.ts

```diff
@@ -130,7 +130,7 @@ function emitL10nError(key: string, bundleName: string, reason: string) {
 }
 
 const createCustomEvent = function (event: string, params: CustomEventInit) {
-  if (window.CustomEvent) {
+  if (typeof window.CustomEvent === 'function') {
```

Comment:
This behavior in IE11 is still slightly different it seems. In the older version, it checks `!window.CustomEvent` and then sets it to `IECustomEvent`, _then_ checks if `window.CustomEvent` is a function before calling it. However if in IE11 `window.CustomEvent` is an Object,, it won't polyfill since `window.CustomEvent` is defined, just is an Object.

In the new code, we check first if the type is `'function'` and polyfill when it's not. Seems like the correct behavior but it is a bit different.